src/student_success_tool/generation/pdp/raw_cohort.py

```diff
@@ -25,7 +26,9 @@ def raw_cohort_record(
 
         # TODO: handle other cases, e.g. gateway course attempted/completed/grades
         record = {
-            "Student GUID": self.student_guid(),
+            "Student GUID": student_guid
```

Comment:
An alternative possible solution would be to use ".unique" here -> `self.unique.student_guid()`

That didn't work, though. My python isn't super strong and so I'm not exactly sure why. I guess when you add a provider that provider doesn't inherit all of the faker's properties?

Reply:
Ha, I was going to suggest exactly this! I'm pretty sure there _is_ a way; maybe it's `self.generator.unique.student_guid()`? I've not had my head in `faker` in a while, and its eccentricities never stick...

Reply:
Good idea, just tried it, but that doesn't work either.
 Another option would be to write my own unique checking in the provider (wouldn't be that hard). We could also bump up the number of digits by a bunch but that wouldn't permenantly solve the problem or anything